beluga/include/beluga/sensor/data/landmark_map.hpp

```diff
@@ -46,9 +47,23 @@ class LandmarkMap {
   /// @brief Constructor.
   /// @param boundaries Limits of the map.
   /// @param landmarks List of landmarks that can be expected to be detected.
-  LandmarkMap(const LandmarkMapBoundaries& boundaries, landmarks_set_position_data landmarks)
+  explicit LandmarkMap(const LandmarkMapBoundaries& boundaries, landmarks_set_position_data landmarks)
       : landmarks_(std::move(landmarks)), map_boundaries_(std::move(boundaries)) {}
 
+  /// @brief Constructor with implicit map boundaries (computed from landmarks).
+  /// @param landmarks List of landmarks that can be expected to be detected.
+  explicit LandmarkMap(landmarks_set_position_data landmarks) : landmarks_(std::move(landmarks)) {
+    if (!landmarks_.empty()) {
+      map_boundaries_.min() = landmarks_[0].detection_position_in_robot;
+      map_boundaries_.max() = landmarks_[0].detection_position_in_robot;
+      for (const auto& landmark : ranges::views::tail(landmarks_)) {
+        const auto& position = landmark.detection_position_in_robot;
+        map_boundaries_.min() = map_boundaries_.min().cwiseMin(position).eval();
+        map_boundaries_.max() = map_boundaries_.max().cwiseMax(position).eval();
```

Comment:
nit^n: Can we avoid a temporary here since the aliasing is harmless?

```suggestion
        map_boundaries_.min() = map_boundaries_.min().cwiseMin(position);
        map_boundaries_.max() = map_boundaries_.max().cwiseMax(position);
```

Reply:
See https://github.com/Ekumen-OS/beluga/pull/419/commits/76f6c1be9e6074ebdf3bcf759d8a5e91434c7e8a.